test/TEST-01-BASIC/create-root.sh

```diff
@@ -1,24 +1,22 @@
 #!/bin/sh
+
+trap 'poweroff -f' EXIT
+
 # don't let udev and this script step on eachother's toes
 for x in 64-lvm.rules 70-mdadm.rules 99-mount-rules; do
     : > "/etc/udev/rules.d/$x"
 done
 rm -f -- /etc/lvm/lvm.conf
 udevadm control --reload
-set -e
-# save a partition at the beginning for future flagging purposes
-sfdisk /dev/sda << EOF
-,1M
-,
-EOF
-
 udevadm settle
-mkfs.ext3 -L '  rdinit=/bin/sh' /dev/sda2
+
+set -ex
+
+mkfs.ext3 -L '  rdinit=/bin/sh' /dev/disk/by-id/ata-disk_root
```

Comment:
is this the double space here  `'  rdinit=/bin/sh'` supposed be there in other words why do you need double space instead of single?

Reply:
part of the test of the kernel command line parsing